sdk/digitaltwins/Azure.DigitalTwins.Core/src/autorest.md

```diff
@@ -22,6 +22,12 @@ Run `generate.ps1` in this directory to generate the code.
 #when generating from official source - The raw link must have a commit hash for C# generator
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/14fb40342c19f8b483e132038f8424ee62b745d9/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json
 
+directive:
```

Comment:
If this works, do we ask ADT to add this to their swagger directly?

Reply:
Also, any indication if this will help other languages?

Reply:
The PR that Pavel linked was directly to autorest.csharp, so this may just be a .NET thing for now

Reply:
Please ask him about other languages, and if we should follow up with others, or if he'll relay this new idea to the other language architects. We either initiate that now, or have to do it again for Java and probably Python.